plugins/yivi-web/translations/en.js

```diff
@@ -1,11 +1,11 @@
 // prettier-ignore
 module.exports = {
-  header: 'Continue with <i class="irma-web-logo">IRMA</i>',
-  helper: 'Can\'t figure it out?<br>Take a look at the <a href="https://irma.app/?lang=en">IRMA website</a>.',
+  header: 'Continue with <i class="yivi-web-logo">Yivi</i>',
+  helper: 'Can\'t figure it out?<br>Take a look at the <a href="https://yivi.app/?lang=en">Yivi website</a>.',
```

Comment:
Is it already clear how the localization will work on yivi.app? Otherwise, we might want to remove the language pinning at all for now and fix that when it is clear.

Reply:
The pinning has been removed; according to my sources, the language will be set by a prefix, i.e.: https://yivi.app/nl/somewhere